catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -329,15 +326,24 @@ private Serializable parseDefaultValue(AttributeDescriptor descriptor, String de
                     List<String> metacardTypes = defaultObj.metacardTypes;
                     if (metacardTypes == null || metacardTypes.isEmpty()) {
                         return Stream.of(() -> {
-                            defaultAttributeValueRegistry.setDefaultValue(attribute, defaultValue);
+                            Dictionary<String, String> props = new Hashtable<>();
+                            props.put("attribute", attribute);
+                            getBundleContext().registerService(DefaultAttributeValue.class,
+                                    new DefaultAttributeValueImpl(attribute, defaultValue),
+                                    props);
                             return true;
                         });
                     } else {
                         return metacardTypes.stream()
                                 .map(metacardType -> (Callable<Boolean>) () -> {
-                                    defaultAttributeValueRegistry.setDefaultValue(metacardType,
-                                            attribute,
-                                            defaultValue);
+                                    Dictionary<String, String> props = new Hashtable<>();
+                                    props.put("attribute", attribute);
+                                    props.put("metacardType", metacardType);
+                                    getBundleContext().registerService(DefaultAttributeValue.class,
```

Comment:
For the case where metacard types are specified, we should be registering a single `DefaultAttributeValue` service using the constructor that takes a `Set<String>`.